LibreNMS/Util/FileCategorizer.php

```diff
@@ -99,6 +102,51 @@ private function validateOs($os)
         return file_exists("includes/definitions/$os.yaml") ? $os : null;
     }
 
+    private function osFromMibs(): void
+    {
+        $mibs = [];
+
+        foreach ($this->items as $file) {
+            if (Str::startsWith($file, 'mibs/')) {
+                $mibs[] = basename($file, '.mib');
+            }
+        }
+
+        $mibs = implode("\n", $mibs);
+
+        if (empty($mibs)) {
+            return;
+        }
+
+        $grep = new Process(
+            [
+                'grep',
+                '--fixed-strings',
+                '--recursive',
+                '--files-with-matches',
+                '--file=-',
+                '--',
+                'includes/definitions/',
+                'includes/discovery/',
+                'includes/polling/',
+                'LibreNMS/OS/',
+                'tests/data/',
```

Comment:
Why did you include the test data and snmprec files?

Reply:
because osFromFile() checks for matches there, but now I realized it only contains oids :) no matches will be found. ill remove